app.js

```diff
@@ -1,8 +1,128 @@
 // import functions and grab DOM elements
+const button1 = document.getElementById('cup-one-button');
+const button2 = document.getElementById('cup-two-button');
+const button3 = document.getElementById('cup-three-button');
+const img1 = document.getElementById('cup-one-img');
+const img2 = document.getElementById('cup-two-img');
+const img3 = document.getElementById('cup-three-img');
+const winsEl = document.getElementById('wins');
+const lossesEl = document.getElementById('losses');
+const totalEl = document.getElementById('total');
+const resetButton = document.getElementById('reset');
+const clearButton = document.getElementById('clear');
+
+const easyButton = document.getElementById('easy');
```

Comment:
Note for future assignments: get in the habit of cleaning up these check failures. These will prevent auto-deployment!